driver/web/bbs_permission_policy.csv

```diff
@@ -0,0 +1,12 @@
+p, get_user_groups, /gr/api/bbs/user/*, (GET), Get User Group BB actions
```

Comment:
When we add a new asset file like this we need to update the `Dockerfile` to make sure it is available once the BB is containerized.